README.md

```diff
@@ -0,0 +1,39 @@
+# Books API
+
+## Database scheme 
+```mermaid
+erDiagram
+Book{
+  long id
+  string title
+  string annotation
+  datetime createdAtUtc
+  string artworkUrl
+  int numberOfCopies
+}
+Status{
+  long id
+  string value
+}
```

Comment:
I suggest that the status field be removed now. Since we are still trying to display the simplest logic in the database, in order for this field to make sense, we need to immediately think about where the employee_id will be and how to link it with other services. Now this status is meaningless without this logic. For now, there is no need to think through the logic with employee, it is easier to remove this field in this iteration.